packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -258,6 +258,8 @@ class CupertinoTextField extends StatefulWidget {
     this.onChanged,
     this.onEditingComplete,
     this.onSubmitted,
+    this.onFocus,
+    this.onBlur,
```

Comment:
I recall a team discussion where it was decided that "unfocus" is superior to "blur".  Should this be "onUnfocus" @HansMuller?